README.md

```diff
@@ -3,74 +3,74 @@
 [![Coverage Status](https://coveralls.io/repos/github/uber/cadence/badge.svg)](https://coveralls.io/github/uber/cadence)
 [![Slack Status](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=social)](http://t.uber.com/cadence-slack)
 
-Visit [cadenceworkflow.io](https://cadenceworkflow.io) to learn about Cadence.
+This repo contains the source code of the Cadence server and other tooling including CLI, schema tools, bench and canary. 
 
-This repo contains the source code of the Cadence server. To implement workflows, activities and worker use [Go client](https://github.com/uber-go/cadence-client) or [Java client](https://github.com/uber-java/cadence-client).
+To implement workflows, use [Go client](https://github.com/uber-go/cadence-client) or [Java client](https://github.com/uber-java/cadence-client) by Cadence, 
```

Comment:
You can implement your workflows with one of our client libraries. The Go and Java libraries are officially maintained by the Cadence team, while the Python and Ruby clients are developed by the community.